common/religion/religions/light_group.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_crime
 	doctrine = doctrine_witchcraft_crime
 
+	#Magic
+	doctrine = doctrine_light_magic_approved
+	doctrine = doctrine_shadow_magic_crime
+	doctrine = doctrine_disorder_magic_crime
+	doctrine = doctrine_order_magic_shunned
+	doctrine = doctrine_life_magic_shunned
+	doctrine = doctrine_death_magic_crime
+	doctrine = doctrine_elemental_spirit_magic_shunned
+	doctrine = doctrine_elemental_decay_magic_shunned
+	doctrine = doctrine_elemental_fire_magic_accepted
```

Comment:
I can't think of a reason that fire magic is accepted instead of ignored.

Reply:
I was thinking of this thing

https://wow.gamepedia.com/Radiant

Reply:
I usually refrain on making in-game playable class mechanics as reference because it would give a lot of conflicting combinations. Most Balance druid trees are dealing arcane damage in-game.
I just shrug them of as the class designer taking liberty to make flavor out of the class ability.

Night Elves Priest dealing Arcane damage:
https://wow.gamepedia.com/Starshards

Disc priest dealing Shadow and Light at once:
https://wow.gamepedia.com/Premonition

Reply:
Fair.
Done!